pyaedt/modules/PostProcessor.py

```diff
@@ -4911,3 +4915,455 @@ def sample_ami_waveform(
+
+        if quantity_name in TOTAL_QUANTITIES:
+            return {i: content[i][0] for i in ["Total", "Unit"]}
+        return {i: content[i][0] for i in ["Min", "Max", "Mean", "Stdev", "Unit"]}
+
+    @pyaedt_function_handler()
+    def evaluate_faces_quantity(
+        self,
+        faces_list,
+        quantity_name,
+        side="Default",
+        setup_name=None,
+        design_variation={},
+        ref_temperature="",
+    ):
+        """Export the field surface output.
+
+        Parameters
+        ----------
+        faces_list : list
+            List of faces to apply.
+        quantity_name : str
+            Name of the quantity to export.
+        side : str, optional
+            Which side of the mesh face to use. The default is ``Default``.
+            Options are ``"Adjacent"``, ``"Combined"``, and ``"Default"``.
+        setup_name : str, optional
+            Name of the setup and name of the sweep. For example, ``"IcepakSetup1 : SteatyState"``.
+            The default is ``None``, in which case the active setup and active sweep are used.
+        design_variation : dict, optional
+            Dictionary of parameters defined for the specific setup with values. The default is ``{}``.
+        ref_temperature: str, optional
+            Reference temperature to use for heat transfer coefficient computation. The default is ``""``.
+
+        Returns
+        -------
+        dict
+            Output dictionary, which depending on the quantity chosen, contains one
+            of these sets of keys:
+
+            - ``"Min"``, ``"Max"``, ``"Mean"``, ``"Stdev"``, and ``"Unit"``
+            - ``"Total"`` and ``"Unit"``
+
+        References
+        ----------
+
+        >>> oModule.ExportFieldsSummary
+        """
+        name = generate_unique_name(quantity_name)
+        self._app.modeler.create_face_list(faces_list, name)
+        fs = self.create_field_summary()
+        fs.add_calculation("Object", "Surface", name, quantity_name, side=side, ref_temperature=ref_temperature)
+        return self._parse_field_summary_content(fs, setup_name, design_variation, quantity_name)
+
+    @pyaedt_function_handler()
+    def evaluate_boundary_quantity(
+        self,
+        boundary_name,
+        quantity_name,
+        side="Default",
+        volume=False,
+        setup_name=None,
+        design_variation={},
+        ref_temperature="",
+    ):
+        """Export the field output on a boundary.
+
+        Parameters
+        ----------
+        boundary_name : str
+            Name of boundary to perform the computation on.
+        quantity_name : str
+            Name of the quantity to export.
+        side : str, optional
+            Side of the mesh face to use. The default is ``"Default"``.
+            Options are ``"Adjacent"``, ``"Combined"``, and ``"Default"``.
+        volume : bool, optional
+            Whether to compute the quantity on the volume or on the surface. The default is ``False``.
```

Comment:
What is the behavior when the default of ``False`` is used? It isn't clear because you would expect the default to be ``True`` or ``False``.